src/ui/page_playback.c

```diff
@@ -367,17 +367,21 @@ void pb_key(uint8_t key) {
     done = false;
     switch (key) {
     case DIAL_KEY_UP: // up
-        if (media_db.cur_sel != (media_db.count - 1)) {
+        if (media_db.cur_sel = (media_db.count - 1)) {
```

Comment:
That should be == otherwise it's an assignment!

Reply:
Oh my, schoolboy error. Fixed